packages/@react-stately/grid/src/GridCollection.ts

```diff
@@ -23,7 +23,7 @@ export class GridCollection<T> implements IGridCollection<T> {
   columnCount: number;
   rows: GridNode<T>[];
 
-  constructor(opts?: GridCollectionOptions<T>) { // TODO what to do with columnCount
+  constructor(opts?: GridCollectionOptions<T>, nodePopulate?: (cell: GridNode<T>) => GridNode<T>) {
```

Comment:
need a better name for the function

Reply:
maybe add the function to the options as a key rather than a separate argument? I think maybe call it `visitNode`?